client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -189,6 +190,18 @@ export const getTask = (
 				actionUrl: `/themes/${ siteSlug }`,
 			};
 			break;
+		case CHECKLIST_KNOWN_TASKS.PROFESSIONAL_EMAIL_MAILBOX_CREATED:
+			taskData = {
+				timing: 2,
+				title: translate( 'Setup your Professional Email' ),
+				description: translate(
+					'Complete your Professional Email setup to start sending and receiving emails from your custom domain today.'
+				),
+				actionText: translate( 'Complete setup' ),
+				isSkippable: false,
+				actionUrl: emailManagement( siteSlug, task.domainName ),
```

Comment:
I've now tested the back end using `domain` as the field for the domain.
```suggestion
				actionUrl: emailManagement( siteSlug, task.domain ),
```

Reply:
Just my thought, but seems to me that domain would be and object with more properties inside, instead of a string. I'm happy to add `task.domain` but for my understand would be better `task.domainName` or `task.domain.name`

Reply:
I agree that `domainName` would be better. But you get to make a quick choice between `domain` and `domain_name` for now. (I don't think I can add `domainName` due to linting/style requirements on the back end.)